sites/funcampus/src/backend/templates/social-networks/blogpost-badges.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n static %}{% spaceless %}
+{% with full_page_url=SITE.web_url|add:page_url %}
+    {% blocktrans with title=page_title asvar mailto_subject %}FUN Campus news: {{ title }}{% endblocktrans %}
+    {% blocktrans with title=page_title|safe url=full_page_url asvar shared_sentence %}FUN Campus news: {{ title }} {{ url }}{% endblocktrans %}
+    <div class="social-network-badges">
+        <a
+            href="https://www.facebook.com/share.php?u={{ full_page_url }}"
```

Comment:
No need to urlencode the `full_page_url` ?
(Multiple occurences of this in this PR)

Reply:
Apparently not and it is already in production on other websites.

Reply:
Ok then :-)